docs/src/tuning_models.md

```diff
@@ -184,9 +184,15 @@ Let's instatiate the tuning strategy:
 ```@repl goof
 latin_minimal = LatinHypercube(nGenerations=2,popSize= 120)
 ```
-Where nGenerations is the number of generations to run the 
-optimisation for and popSize is the population size in 
-the genetic algorithm. 
+Where `nGenerations` is the number of generations to run the 
+optimisation for and `popSize` is the population size in 
+the genetic algorithm. Pay attention that these parameters
+have nothing to do with the ones specified in the `TunedModel` 
```

Comment:
Much better.  Instead of "ones specified in" with "the iteration parameter `n` in".